context/parseLambda.ts

```diff
@@ -4,6 +4,7 @@ import {
   validateAddress,
   ValidationResult,
 } from "@taquito/utils";
+import { parse } from "path";
```

Comment:
It looks like an auto-import from the IDE that is unused

Reply:
removed